README.md

```diff
@@ -1,4 +1,4 @@
-# Bucketeer CLI &nbsp;[![Build Status](https://travis-ci.com/UCLALibrary/bucketeer-cli.svg?branch=main)](https://travis-ci.com/UCLALibrary/bucketeer-cli)
+# Bucketeer CLI
```

Comment:
GitHub Actions has a badge too. Here is Fester's for an example:

<pre>[![Maven Build](https://github.com/uclalibrary/fester/workflows/Maven%20PR%20Build/badge.svg)](https://github.com/UCLALibrary/fester/actions)</pre>

The name of the workflow comes from the name in the YAML file.